pkg/kudoctl/util/repo/testdata/zk-crd-golden1/operatorversion.golden

```diff
@@ -15,9 +15,11 @@ spec:
     - name: cpus
       description: Amount of cpu to provide to Zookeeper pods
       default: "0.25"
+      required: true
     - name: memory
       description: Amount of memory to provide to Zookeeper pods
       default: "1Gi"
+      required: true
```

Comment:
I don't see how any value with a default is a required  field.   It has me question why "default" and "required" .  If there  is a default it isn't required.   If there is no default it is required.   Is that sufficient?  or am I missing something?

Reply:
yeah, you're right. I think I added this because the golden tests are comparing the yaml 1:1 ... but in real life yes, you don't need to specify that at all, required is true by default all the time, not just when default is specified